satellite/audit/service.go

```diff
@@ -31,7 +31,10 @@ type Config struct {
 	MinDownloadTimeout time.Duration `help:"the minimum duration for downloading a share from storage nodes before timing out" default:"25s"`
 	MaxReverifyCount   int           `help:"limit above which we consider an audit is failed" default:"3"`
 
-	Slots int `help:"number of reservoir slots allotted for nodes, currently capped at 2" default:"1"`
+	ChoreInterval     time.Duration `help:"how often to run the reservoir chore" default:"24h"`
+	QueueInterval     time.Duration `help:"how often to recheck an empty audit queue" default:"30s"`
```

Comment:
if the audit queue is empty, we will need to wait for the chore to run again for it to fill up. Since the chore interval is 24h, I feel like 30s is too frequent for the queue interval. I think it should be at least 1h by default.